### PR TITLE
Carry 15457: systemctl enable and start docker

### LINKED ISSUE #15456: The CentOS installation instructions should use systemctl and both start and enable docker
The CentOS installation instructions at https://docs.docker.com/docker/installation/centos/ (`docs/installation/centos.md`) suggest that after installing the package one should:

```
4. Start the Docker daemon.

                $ sudo service docker start
```

`service` is a thin layer on top of systemctl on CentOS.  And starting a systemctl service is only half the job.  After a reboot, the service will be stopped again.  To make it run on every boot, it needs to be "enabled" as well:

```
    sudo systemctl enable docker
    sudo systemctl start docker
```

### PR DESCRIPTION
**Pull request by thaJeztah**
Carry of 15457; fixes the indentation, caused by mixed tabs/spaces.

closes #15457
fixes #15456

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @moxiegirl

File: docs/installation/centos.md [hunk 1]
```diff
@@ -58,9 +58,10 @@ There are two ways to install Docker Engine.  You can use `curl` with the  `get.
 		
 		This script adds the `docker.repo` repository and installs Docker.
 
-4. Start the Docker daemon.
+4. Start the Docker daemon and enable it so it starts on future boots.
 
-		$ sudo service docker start
+		$ sudo systemctl start docker
+		$ sudo systemctl enable docker
```
Comment (moxiegirl) [inline docs/installation/centos.md]: How is this different that "Start the docker daemon at boot" this heading down the page?  Why is the command different. I'd prefer to move the start on boot to its own section...

**Comment by moxiegirl** [comment]
ping @thaJeztah

**Comment by thaJeztah** [comment]
Oh, sorry, this one slipped my attention. I'll have a look when I'm home; tbh I carried, just to fix the indentation (that's what the previous PR stalled on ;-))

**Comment by icecrime** [comment]
Ping @thaJeztah!

**Comment by thaJeztah** [comment]
I'll close this one for now; docs need more checks for systemd related changes and I needed to do some testing :)

**Pull request closed without merging**